lxd/db: Add missing unique key

stgraber opened the pull request:
Signed-off-by: Stéphane Graber <stgraber@ubuntu.com>

freeekanayaka commented:
This is missing ```make update-schema```. Also, I guess it means that people tracking edge will not get the constraint. A different update version that introduces the constraint would be needed, or? YMMV

stgraber commented:
doh, I was about to finish addressing the feedback when @brauner merged. Going to send a follow-up with the bits I didn't get to push prior to the merge.

stgraber commented:
https://github.com/lxc/lxd/pull/6614
